cloudslang-cli/src/main/java/io/cloudslang/lang/cli/services/SyncTriggerEventListener.java

```diff
@@ -85,7 +89,20 @@ public synchronized void onEvent(ScoreEvent scoreEvent) throws InterruptedExcept
             case ScoreLangConstants.EVENT_OUTPUT_END:
                 Map<String, Serializable> outputs = extractOutputs(data);
 
```

Comment:
with this new functionality, I would rewrite this section little bit: test whether the event refers to a task or flow end and assign behavior accordingly (for DRY - now you are calling `extractNotEmptyOutputs` twice for each task). Thay way you would only one `extractOutputs method`. WDYT?